tests/Utilities/constants.py

```diff
@@ -5,6 +5,9 @@
 timedelta2 = timedelta(days=2)
 timedelta8 = timedelta(days=8)
 
+name1 = "apple"
```

Comment:
Why did you decide to name these like this?

Reply:
I figured when our users want to remove a `Food` item, they are going to refer to them by name, rather then passing in an entire `Food` object. Since our `remove` function takes in a `food_name`, I just decided to separate out the name into a variable to make testing easier.

You can see where it's used in `testRemoveFoods` in `test_foodStorage.py`